frontend/src/main/scala/bloop/cli/Commands.scala

```diff
@@ -46,9 +46,8 @@ object Commands {
       @ExtraName("p")
       @HelpMessage("The project to test.")
       project: String,
-      @ExtraName("all")
-      @HelpMessage("Run the tests in dependencies. Defaults to true.")
-      aggregate: Boolean = false,
+      @HelpMessage("Do not run tests for dependencies. By default, false.")
```

Comment:
Do you think it fails weird? It sounds super natural to me because it's an imperative, and I like descriptions that say exactly what they do without if conditions in the description, otherwise it becomes too verbose. Your suggestion was actually the first version i had but i removed it because i wasn't satisfied :smile:

Reply:
Okay 👍